src/Illuminate/Foundation/Testing/Concerns/InteractsWithRedis.php

```diff
@@ -36,14 +36,10 @@ public function setUpRedis()
 
         if (! extension_loaded('redis')) {
             $this->markTestSkipped('The redis extension is not installed. Please install the extension to enable '.__CLASS__);
-
-            return;
         }
 
         if (static::$connectionFailedOnceWithDefaultsSkip) {
             $this->markTestSkipped('Trying default host/port failed, please set environment variable REDIS_HOST & REDIS_PORT to enable '.__CLASS__);
-
-            return;
```

Comment:
Why?

Reply:
`markTestSkipped()` has `@psalm-return never-return` annotation. There are 48 occurrences of this method in Laravel tests and only those two are followed by a return.